docs/proposals/network/README.md

```diff
@@ -0,0 +1,66 @@
+# Software Defined Network
+
+## Analysis
+
+### Examples
+
+- Canal
+- Calico
+- Flannel
+- Weave
+- Cilium
+
+![benchmark-cni](https://miro.medium.com/v2/resize:fit:1400/1*zDiFQZVXuL923x0LbmXMaA.png)
```

Comment:
Pretty nice. Keep in mind that simplicity and is one of our goals here, performance is not important for now.

Reply:
Yes, of course, we need something that is working first

We keeped the examples section